### ISSUE TITLE
display xliff <note> tag more visible

### ISSUE THREAD

**Issue by transl8bzimport** [translate/virtaal#2076]
_Version: 0.7.0_

_Originally posted by Adam:_

_Created [attachment 796](http://bugs.locamotion.org/attachment.cgi?id=796)_

file used for testing.

Currently the <note>Blub</note> gets displayed only, if hovering an item in the list. Above the translation, only the id is displayed.

It would be better to display the note, if it is available, because the id is often not as important to the translator.

<trans-unit id="1" restype="caption">
   <note>Blub</note>
   <source>Titel</source>
</trans-unit>

**Comment by Abekonge** [comment]
> Does it make sense to view all comments as read only and shown above the source text?

Yes I think that would make a better solution. It could work as a sort of fallback, even though you have an ideal with different sort of comments in different places - it would make it much more useful for us.

For out workflow we extract the XLIFF with Angular 2's i18n tool, which does not source the comments. For us it would be a hassle to add comment-origins in the XLIFF afterwards.
